[DCOS-50378] Release notes for HDFS 2.6.0-3.2.0 (#2486)
mesosphere/dcos-docs-site: pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/release-notes/index.md

---
layout: layout.pug
navigationTitle: Release Notes
excerpt: Discover the new features, updates, and known limitations in this release of the HDFS Service
title: Release Notes
menuWeight: 0
model: /mesosphere/dcos/services/hdfs/data.yml
render: mustache
---

# Release Notes for HDFS Service version 2.6.0-3.2.0

> 

Reviewer: ```suggestion

## Updates
- Updated HDFS version to `3.2.0`. Note: our package no longer includes Cloudera's `cdh` variant
- Updated the SDK to version `0.56.1` 
- Oracle JDK is replaced by OpenJDK 8
```

Author: Done

## Feature

> - HDFS cluster can be accessed from outside the DC/OS cluster by setting up an external DNS. This feature can be utilized through `custom_domain` property.

Reviewer: ```suggestion
- Added support for custom domains. See <add link to custom domain section here>
- Added support for DC/OS Storage Service (DSS). See [DSS documentation](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/)
- Added marathon service scheduler checks
```

Reviewer: Please add a section in HDFS docs (could be security) similar to this: https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/security/#forwarding-dns-and-custom-domain

That way users can reference what to do with custom domains

Author: Done

## Bug Fixes

> - A bug in Apache Derby has been fixed here temporarily. Bug referred to miscalculation of number of bytes read for an principal entry in the Kerberos keytab file. This is addressed in this PR: ([#44](https://github.com/apache/directory-kerby/pull/44)). There is also a ticket in Apache Hadoop repository: ([#16283](https://issues.apache.org/jira/browse/HADOOP-16283)).

Reviewer: ```suggestion
- Fixed a bug in Apache Derby related to the miscalculation of bytes read for a principal entry in the Kerberos keytab file. See [HADOOP-16283](https://issues.apache.org/jira/browse/HADOOP-16283)).
```

Author: Done

## Improvements

> - rollingUpgrade feature of Hdfs is used to upgrade the framework to latest Apache HDFS version.

Reviewer: You can delete this. Not really an improvement as it was more necessary to complete the upgrade process. Not needed to share with users as it's something under the hood.

Author: Done

## Updates

> - Hadoop tar ball has been updated with snappy library.

Reviewer: Please move this to the above Updates section including the version of snappy.

Author: Done

navigationTitle: Release Notes
excerpt: Discover the new features, updates, and known limitations in this release of the HDFS Service
title: Release Notes
menuWeight: 0
model: /mesosphere/dcos/services/hdfs/data.yml
render: mustache
---

# Release Notes for HDFS Service version 2.6.0-3.2.0

> ## Feature

Reviewer: ```suggestion
## Features
```

Author: Done

- Updated HDFS version to `3.2.0`. Note: our package no longer includes Cloudera's `cdh` variant

> - Updated Snappy library to version `1.1.4` in the Hadoop tar ball.

Reviewer: ```suggestion
- Updated Snappy library to version `1.1.4`.
```

Author: Okay

- Added marathon service scheduler checks

## Bug Fixes
- Fixed a bug in Apache Derby related to the miscalculation of bytes read for a principal entry in the Kerberos keytab file. See [HADOOP-16283](https://issues.apache.org/jira/browse/HADOOP-16283)).

> ## Updates

Reviewer: Can you move Updates above Features? It follows the same pattern as our other packages e.g
- cassandra: https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/release-notes/
- kafka: https://docs.d2iq.com/mesosphere/dcos/services/kafka/2.6.0-2.2.1/release-notes/

Author: Done

- Updated HDFS version to `3.2.0`. Note: our package no longer includes Cloudera's `cdh` variant

> - Hadoop tar ball has been updated with Snappy 1.1.4 library.

Reviewer: ```suggestion
- Updated Snappy library to version `1.1.4`
```

Author: Done

- Updated the SDK to version `0.56.1` 

> - Oracle JDK is replaced by OpenJDK 8

Reviewer: ```suggestion
- Oracle JDK is replaced with OpenJDK 8
```

Author: Done
